src/jobs/jobs.service.ts

```diff
@@ -51,10 +46,18 @@ export class JobsService {
 
   async findAll(
     filter: IFilters<JobDocument, FilterQuery<JobDocument>>,
+    user: JWTUser
   ): Promise<JobClass[]> {
-    const whereFilters: FilterQuery<JobDocument> = filter.where ?? {};
+    var whereFilters: FilterQuery<JobDocument> = filter.where ?? {};
     const { limit, skip, sort } = parseLimitFilters(filter.limits);
 
+    if (user.username !== "admin") {
```

Comment:
We talked with @despadam and decided to think about it after GET tests. Because CASL should cover correct permissions but the data instance authorization is never checked for GET endpoints, permissions are granted based solely on the endpoint authorization.

Reply:
Done in https://github.com/SciCatProject/scicat-backend-next/pull/1273/commits/749edd95e0b4bfb0e5276c18ce594740f48a9a36